### PR TITLE
Admin projects styles

### PR DESCRIPTION
**Pull request by gtenaschuk**

### CODE REVIEW

**Comment by mustela** [comment]
I would move the project name into the square with all the other details. And you could put the project name in the breadcrumb as we are doing in other places, like dashboards. 

Screenshot 2022-02-23 at 16 50 20

**Comment by mustela** [comment]
Also, could we also update this listing and details view? /admin/teams

**Review by mustela — COMMENTED** [review]
Just left you a few comments, but looking great!

File: lib/live_sup_web/live/admin/project_live/show.ex [hunk 1]
```diff
@@ -2,6 +2,7 @@ defmodule LiveSupWeb.Admin.ProjectLive.Show do
   use LiveSupWeb, :live_view
 
   alias LiveSup.Core.Projects
+  alias LiveSup.Schemas.Project
```
[the remark this answers is not in the document]
Comment (gtenaschuk) [reply lib/live_sup_web/live/admin/project_live/show.ex:5]: Was the way i found to get the avatar, is there another way to get it from the project object directly?

File: lib/live_sup_web/live/admin/project_live/header_component_show.ex [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <div class="justify-between flex flex-row space-x-4 border-b dark:border-primary-darker px-4 py-2">
+      <div class="flex items-center basis-6/12">
+        <%= link "Home", class: "text-blue-500 after:mx-4  after:content-['>'] dark:after:text-white", to: Routes.home_path(LiveSupWeb.Endpoint, :index) %>
+        <%= link "Admin", class: "text-blue-500 after:mx-4  after:content-['>'] dark:after:text-white", to: Routes.home_path(LiveSupWeb.Endpoint, :index) %>
+        <%= link "Projects", class: "text-blue-500 after:mx-4  after:content-['>'] dark:after:text-white", to: Routes.admin_project_index_path(LiveSupWeb.Endpoint, :index) %>
+        <span>Project Details</span>
+      </div>
+
+      <div class="flex justify-end basis-6/12 text-right items-center">
+        <%= AddButtonComponent.render(%{path: Routes.admin_project_index_path(LiveSupWeb.Endpoint, :new)}) %>
```
[the remark this answers is not in the document]
Comment (gtenaschuk) [reply lib/live_sup_web/live/admin/project_live/header_component_show.ex:15]: Yeah i need to finish this, I created the PR to have a way to make a review on these code :)

**Comment by gtenaschuk** [comment]
I will work on these changes

File: lib/live_sup_web/live/admin/project_live/show.ex (hunk 1 above)
[the remark this answers is not in the document]
Comment (mustela) [reply lib/live_sup_web/live/admin/project_live/show.ex:5]: Sorry! Didn't see how you were using it. It's perfect!

**Pull request merged**